### ISSUE TITLE
Accessibility - no alt text on image links

### ISSUE THREAD

**Issue by edent** [IBMDeveloperUK/IBMDeveloperUK.github.io#32]
None of the images on the site appear to have labels. This is particularly bad on the links under each person's photo.  How is a visually impaired user expected to know where the links go?

![Screenshot showing lack of alt text on image links](https://user-images.githubusercontent.com/837136/95185077-2c1ac800-07c0-11eb-851e-f45aa09e043d.png)
